docs/apache-airflow/executor/kubernetes.rst

```diff
@@ -170,6 +164,29 @@ Additionally, the Kubernetes Executor enables specification of additional featur
 .. @enduml
 .. image:: ../img/k8s-happy-path.png
 
+Comparison with CeleryExecutor
+------------------------------
+
+In contrast to CeleryExecutor, KubernetesExecutor does not require additional components such as Redis and Flower, but does require access to Kubernetes cluster.
+
+With KubernetesExecutor, each task runs in its own pod. The pod is created when the task is queued, and terminates when the task completes.
+Historically, in some cases this presented a resource utilization advantage over CeleryExecutor, where you needed a fixed number of
+long-running celery worker pods, whether or not there were tasks to run.
+
+However, the official Apache Airflow helm chart can automatically scale celery workers down to zero based on the number of tasks in the queue,
+so when using the official chart, this is no longer an advantage.
+
+With Celery workers you will tend to have less task latency because the worker pod is already up and running when the task is queued.  On the
```

Comment:
```suggestion
With Celery workers you will tend to have less task latency because the worker pod is already up and running when the task is queued. On the
```